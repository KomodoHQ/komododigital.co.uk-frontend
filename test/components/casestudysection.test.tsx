import React from 'react';
import CaseStudySection from '../../src/components/casestudysection';
import renderer from 'react-test-renderer';

test('CaseStudySection element is rendered', () => {
  const data = {
  };
  const component = renderer.create(<CaseStudySection data={data} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
