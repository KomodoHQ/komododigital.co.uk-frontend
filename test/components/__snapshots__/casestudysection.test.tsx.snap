// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CaseStudySection element is rendered 1`] = `
<div>
  CaseStudySection
</div>
`;
